.github/workflows/nightlies.yml

```diff
@@ -63,3 +63,23 @@ jobs:
             --user scipy-wheels-nightly \
             --skip-existing \
             dist/matplotlib-*.whl
+
+      - name: Remove old uploads to save space
+        shell: bash
+        run: |
+          N_LATEST_UPLOADS=5
+
+          # Remove all _but_ the last "${N_LATEST_UPLOADS}" package versions
+          # N.B.: `anaconda show` places the newest packages at the bottom of the output
+          # of the 'Versions' section and package versions are preceded with a '   + '.
+          anaconda show scipy-wheels-nightly/matplotlib &> >(grep '+') | \
+              sed 's/.* + //' | \
```

Comment:
This is somewhat brittle as it relies on user facing stdout not changing, instead of consuming API output. Though this seems about as good/bad as trying to use

```console
$ python -m pip index \
    --index-url https://pypi.anaconda.org/scipy-wheels-nightly/simple \
    --pre \
    versions matplotlib | \
    grep 'Available versions'
WARNING: pip index is currently an experimental command. It may be removed/changed in a future release without prior warning.
Available versions: 3.6.0.dev2573+g3eadeacc06, 3.6.0.dev2569+g3522217386, 3.6.0.dev2553+g3245d395d9
```

to get the versions available.